zonemaster/zonemaster-gui
# Swedish translation

## Purpose

Adds Swedish translation for missing items.

## Changes

Updates `src/assets/i18n/sv.json`.

## How to test this PR

Select Swedish, select a domain name to test, fold out "options, and verify that all error messages about missing information is in Swedish.
